docs/sources/crd.md

```diff
@@ -2,50 +2,50 @@
 
 CRD source provides a generic mechanism to manage DNS records in your favourite DNS provider supported by external-dns.
 
-### Details
+## Details
 
 CRD source watches for a user specified CRD to extract [Endpoints](https://github.com/kubernetes-sigs/external-dns/blob/HEAD/endpoint/endpoint.go) from its `Spec`.
 So users need to create such a CRD and register it to the kubernetes cluster and then create new object(s) of the CRD specifying the Endpoints.
 
-### Registering CRD
+## Registering CRD
 
 Here is typical example of [CRD API type](https://github.com/kubernetes-sigs/external-dns/blob/HEAD/endpoint/endpoint.go) which provides Endpoints to `CRD source`:
 
 ```go
 type TTL int64
 type Targets []string
 type ProviderSpecificProperty struct {
-	Name  string `json:"name,omitempty"`
-	Value string `json:"value,omitempty"`
+ Name  string `json:"name,omitempty"`
```

Comment:
This not seem correct. Let me know if you wanna try to fix in follow-up. This suppose to be a tab with intendation 4.

Reply:
Good catch ! That was unintended, I fixed it in my last commit.